src/main/java/io/vertx/ext/jdbc/impl/actions/JDBCTypeWrapper.java

```diff
@@ -0,0 +1,198 @@
+package io.vertx.ext.jdbc.impl.actions;
```

Comment:
For Eclipse Compliance, can you add a copyright header just like the other files?